mino/minogrpc/server.go

```diff
@@ -379,20 +403,28 @@ func (s sender) Send(msg proto.Message, addrs ...mino.Address) error {
 		sendMsg := &OverlayMsg{
 			Message: envelopeAny,
 		}
-
 		err = player.streamClient.Send(sendMsg)
+		if err == io.EOF {
+			return nil
+		}
 		if err != nil {
 			return xerrors.Errorf("failed to call the send on client stream: %v", err)
 		}
 	}
 
+	if !ok {
```

Comment:
In the case of a stream, it's fair to return an error if something bad happened I guess. You could use a go routine per address in the for loop and gather possible error/nil in an array then create an error from the list if any.

Reply:
but in this case this *could* be a normal behaviour. I am displaying a warning because this is very likely to be an undesirable situation, but I can imagine situations where it is normal that no senders are included in the list of participants.

Reply:
I'm not sure to get the purpose. It will warn if none of the address matches but if you provide a wrong address, it will silently ignore and you have no way to know one was wrong. You can check the length at the beginning.

Reply:
Then I could add another warning at line 383 ?  I did not because excluding someone that is not in the list of address is a normal behavior, while excluding everyone is an orange flag.

Reply:
Thus my point of collecting the errors that are happening in each iteration and then make a summary of the results.

Reply:
Ok. Changing the Send interface to return a `chan error`